data/core.yaml

```diff
@@ -1812,6 +1812,11 @@ en:
           one: "{highway} is disconnected from other roads and paths"
           other: "{count} routable features are connected only to each other"
         reference: "All roads, paths, and ferry routes should connect to form a single routing network. This feature (or small collection of features) is either disconnected from the routing network or only connected to it via a feature that few applications consider routable."
+    empty_start_date:
+      title: Empty start date
+      feature:
+        message: "Start date shouldn't be empty"
+      reference: "Features should have a start date"
```

Comment:
I changed the text to explain the reason better. I also tried to add a dynamic fix to focus on the EDTF date field, but i couldn't figure out how. It can be done using document.getElementById(), but I believe that doesn't fit the project's style properly. There should be an easy way to do it using `context.ui()`, I will keep trying for a while